packages/fxa-content-server/app/scripts/models/user.js

```diff
@@ -21,14 +21,24 @@ const ResumeTokenMixin = require('./mixins/resume-token');
 const UrlMixin = require('./mixins/url');
 const Storage = require('../lib/storage');
 const vat = require('../lib/vat');
+const ProfileClient = require('../lib/profile-client');
+const MarketingEmailClient = require('../lib/marketing-email-client');
+const InterTabChannel = require('../lib/channels/inter-tab');
 
 var User = Backbone.Model.extend({
   initialize (options = {}) {
+    this._config = options.config;
```

Comment:
Can you set `this._config = options.config || {}` to avoid all kinds of unit test breakage?